src/themes/default/pages/Checkout.vue

```diff
@@ -310,14 +310,21 @@ export default {
     display: block;
   }
   .number-circle {
-    width: 40px;
-    height: 40px;
+    width: 35px;
+    height: 35px;
   }
-  .section-disabled {
-    cursor: not-allowed;
-  }
-  .section-editable {
-    cursor: pointer;
+  .line {
+    &:after {
+      content: '';
+      display: block;
+      position: absolute;
+      top: 0;
+      left: 17px;
+      z-index: -1;
+      width: 1px;
+      height: 100%;
+      background-color: #f2f2f2;
```

Comment:
you should import the color variables here from `global_vars.scss`

Reply:
Done